features/utils.py

```diff
@@ -51,7 +51,7 @@ def init_remote_firefox_browser():
     profile.set_preference('network.negotiate-auth.trusted-uris', 'https://')
     op = Options()
     op.profile = profile
-    op.add_argument("-headless")
+    #op.add_argument("-headless")
```

Comment:
Instead of removing this, we could use the `CI` environment variable so it headless on the pipeline to save on resources
```suggestion
    if 'CI' in os.environ:
        op.add_argument("-headless")
```

Reply:
Thanks. Updated.